tech-specs/src/explore/design/ibc.md

```diff
@@ -0,0 +1,127 @@
+# IBC integration
+
+[IBC](https://arxiv.org/pdf/2006.15918.pdf) allows a ledger to track another ledger's consensus state using a light client. IBC is a protocol to agree the consensus state and to send/receive packets between ledgers.
+
+## Transaction for IBC
+A requester (IBC relayer or user) who wants to execute IBC operations on a ledger sets IBC packet or message like `MsgCreateAnyClient` as transaction data and submit the following transaction. IBC validity predicate is invoked after this transaction execution to check the IBC operation is validated. The trigger to invoke IBC validity predicate is `insert_verifier()` in IBC-related operation to add an address for IBC `InternalAddress::Ibc`.
```

Comment:
Well, as long as state owned by the IBC validity predicate is written, the VP should be triggered, right?

Reply:
That's correct. `changed_keys` should have IBC-related keys after IBC-related transactions are executed.
I modified that.